src/fetch_tse_data.py

```diff
@@ -0,0 +1,254 @@
+import shutil
+import pandas as pd
+import numpy as np
+import os
+import urllib
+import zipfile
+import glob
+
+
+
+FILENAME_PREFIX='consulta_cand_'
+TEMP_PATH = '../data/tse_temp'
+TSE_CANDIDATES_URL='http://agencia.tse.jus.br/estatistica/sead/odsele/consulta_cand/'
+OUTPUT_DATASET_PATH = '../data/2017-03-31-tse-candidates.xz'
+os.makedirs(TEMP_PATH)
+
+# setting year range from 2004 to 2016. this will be modified further to 'from 1994 to 2016'
+year_list=[str(year) for year in (range(2004,2017,2))]
+
+# Download files
+for year in year_list:
+    filename=FILENAME_PREFIX+year+'.zip'
+    file_url=TSE_CANDIDATES_URL+filename
+    output_file=os.path.join(TEMP_PATH,filename)
+    urllib.request.urlretrieve(file_url,output_file)
+
+# Unzip downloaded files
+for year in year_list:
+    filename=FILENAME_PREFIX+year+'.zip'
+    filepath=os.path.join(TEMP_PATH,filename)
+    zip_ref = zipfile.ZipFile(filepath, 'r')
+    zip_ref.extractall(TEMP_PATH)
+    zip_ref.close()
+
+
+# ### Adding the headers
+# The following headers were extracted from LEIAME.pdf in consulta_cand_2016.zip.
+header_consulta_cand_till2010=[
```

Comment:
Considering this headers section:

- [ ] We don't need the `#` with no comments at the end of the lines
- [ ]  Also I don't think we need the headers you skip with the `#`
- [ ] And we need to translate the headers to English ([example](https://github.com/datasciencebr/serenata-toolbox/blob/master/serenata_toolbox/ceap_dataset.py#L51-L151))